syntax/blocks.js

```diff
@@ -155,7 +155,7 @@ function loadLanguage(code, language) {
     var m = iconPat.exec(spec)
     if (m) {
       var image = m[0]
-      var hash = nativeHash.replace(image, unicodeIcons[image])
+      var hash = nativeHash.replace(hashSpec(image), unicodeIcons[image])
```

Comment:
`nativeHash` is a return value of `hashSpec` and `@XXXX` doesn't include upper letters.
But, `image` includes upper letters like `@greenFlag`, so we have to apply `hashSpec` to `image`.